tests/integration/test_corecli.py

```diff
@@ -0,0 +1,44 @@
+import json
+import os
+
+from .common import exec_cmd, default_cluster, default_cluster_with_plugins  # noqa: F401
+
+import pytest
+
+
+@pytest.mark.skipif(os.environ.get('DCOS_TEST_CORECLI') is None, reason="no core CLI bundle")
```

Comment:
Not `DCOS_CLI_TEST_CORECLI`? `CLI` would then be redundant but the variable would have a name aligned with `DCOS_CLI_EXPERIMENTAL_AUTOINSTALL_PLUGINS`.

Reply:
This is consistent with https://github.com/dcos/dcos-cli/blob/master/tests/launch_cluster.py#L56-L59. That being said I wouldn't mind changing these to add `CLI`, but it should be done separately.